desktop/webpack.config.base.js

```diff
@@ -16,8 +16,8 @@ module.exports = {
       exclude: /(node_modules|\/dist\/)/,
       query: {
         cacheDirectory: true,
-        plugins: ['transform-runtime'],
-        presets: ['es2015', 'stage-1', 'react']
+        plugins: ['babel-plugin-transform-runtime'].map(require.resolve),
```

Comment:
@chrisnojima IIRC Babel uses the location of `.babelrc` as the base to search for modules. Otherwise it looks relative to the file. So I get an error for both entry points like:

```
ERROR in ../shared/test/render-dumb-sheet.js
Module build failed: ReferenceError: Unknown plugin "transform-runtime" specified in "base" at 0, attempted to resolve relative to "/Users/mgood/Documents/dev/keybase/client/shared/test"
[...]

ERROR in ../shared/more/dumb-component-map.js
Module build failed: ReferenceError: Unknown plugin "transform-runtime" specified in "base" at 0, attempted to resolve relative to "/Users/mgood/Documents/dev/keybase/client/shared/more"
[...]
```

I'm not sure why this isn't happening for other webpack entry points, any ideas? Could also solve this by adding a `.babelrc` in the places this config is used (which might need to be duplicated in `shared`, which is why I went for this approach). `.babelrc` would be cleaner, though.